### PR TITLE
Adding weather command

### PR DESCRIPTION
**Pull request by JusteenR**
[Issue](https://github.com/ACM-VIT/golly-bot/issues/20)

Created a command `!weather <location_city>,<location_country>`, and the bot replies with weather information for that location using  [OpenWeatherMap GO API](https://github.com/briandowns/openweathermap). Returns weather in English and Fahrenheit. To use the OpenweatherMap API, you need to obtain an API key. Sign up [here](http://home.openweathermap.org/users/sign_up) and insert it in the code where indicated.

image

### CODE REVIEW

**Comment by JusteenR** [comment]
@z404 Not sure how to add you as a reviewer but would appreciate it if you could look over my changes.

File: main.go [hunk 1]
```diff
@@ -163,6 +164,19 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 					return
 				}
 			}
+		case botPrefix + "weather":
+			var apiKey = "your apikey here"
```
Comment (z404) [inline main.go:168]: Small change: Name the apiKey variable aptly (include openweather in the name) so we know what apiKey this is. Define the api key at the start of main.go, along with the discord bot token so we remember to put this in the env file when working on issue #7

**Review by z404 — CHANGES_REQUESTED** [review]

**Comment by z404** [comment]
Looks great and works perfectly, just a few slight changes and you're good to go :D

File: main.go (hunk 1 above)
Comment (JusteenR) [reply main.go]: Let me know if the changes are good

**Review by z404 — APPROVED** [review]
LGTM! Good work :D

**Pull request merged**